examples/gitlab/gitlab-push-listener.yaml

```diff
@@ -60,6 +49,9 @@ spec:
             eventTypes:
               - Push Hook  # Only push events
       bindings:
-        - ref: gitlab-push-binding
+        - name: gitrevision
+          value: $(body.checkout_sha)
+        - name: gitrepositoryurl
+          value: $(body.repository.git_http_url)
```

Comment:
I see both `ref` and `name, value` are allowed together for binding, So can we update example to show such scenario?
something like below
```
      bindings:
        - ref: gitlab-push-binding
        - name: gitrevision
          value: $(body.pull_request.head.sha)
```

```
apiVersion: triggers.tekton.dev/v1alpha1
kind: TriggerBinding
metadata:
  name: gitlab-push-binding
spec:
  params:
    - name: gitrepositoryurl
      value: $(body.repository.git_http_url)
```

Reply:
Sure, I'm planning on changing all of our examples to use embedded bindings by default in a PR once this is merged. I'd prefer to create a new example to show that refs and embeds can be mixed and matched instead of adding it to this one example. What do you think of doing that in the followup PR?

Reply:
Sure :+1: